tests/test_fields.py

```diff
@@ -848,6 +849,40 @@ class TestNoOutputFormatDateField(FieldValues):
     field = serializers.DateField(format=None)
 
 
+class FakeTimezone(datetime.tzinfo):
+
+    def __repr__(self):
+        return "<FakeTimezone>"
+
+    def utcoffset(self, dt):
+        return datetime.timedelta(1)
+
+    def tzname(self, dt):
+        return "FakeTimezone"
+
+    def dst(self, dt):
+        return datetime.timedelta(1)
+
+
+class TestAwareDateTimeField:
+
+    @override_settings(USE_TZ=True)
+    def test_with_timezone_active(self):
+        naive_now = timezone.make_naive(timezone.now(), timezone.UTC())
+        timezone.activate(FakeTimezone())
+        field = serializers.DateTimeField(default_timezone=timezone.UTC())
+        aware_now = field.enforce_timezone(naive_now)
+        assert aware_now.tzname() == 'FakeTimezone'
+        timezone.deactivate()
```

Comment:
Would it make sense to have tests that demonstrate the effect on the field input and output?
As it stands Ii's unclear to me exactly what these are testing.